### ISSUE TITLE
Strange error while attempting to debug test

### ISSUE THREAD

**Issue by firelizzard18** [dotnet/vscode-csharp#1960]
`dotnet --info` output:
```
.NET Command Line Tools (2.1.2)

Product Information:
 Version:            2.1.2
 Commit SHA-1 hash:  5695315371

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.16299
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.1.2\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.3
  Build    : a9190d4a75f4a982ae4b4fa8d1a24526566c69df
```

- VSCode Version: Code 1.19.2 (490ef761b76b3f3b3832eff7a588aac891e5fe80, 2018-01-10T15:55:03.538Z)
- OS Version: Windows_NT x64 10.0.16299
- C# Extension version: 1.13.1

I'm having trouble debugging a test in VSCode. I deleted `bin` and `obj` and tried debugging via code lens again (i.e. clicking on "debug test" above the method) and got the following message:

```
Failed to start debugger: "System.NullReferenceException: Object reference not set to an instance of an object.
 at OmniSharp.DotNetTest.TestManager.SendMessage[T](String messageType, T payload) in C:\\projects\\omnisharp-roslyn\\src\\OmniSharp.DotNetTest\\TestManager.cs:line 225
 at OmniSharp.DotNetTest.VSTestManager.<DiscoverTestsAsync>d10.MoveNext() in C:\\projects\\omnisharp-roslyn\\src\\OmniSharp.DotNetTest\\VSTestManager.cs:line 249
 --- End of stack trace from previous locat
```

My project:
```
<Project Sdk="Microsoft.NET.Sdk">
  <PropertyGroup>
    <TargetFramework>net462</TargetFramework>

    <IsPackable>false</IsPackable>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0" />
    <PackageReference Include="xunit" Version="2.3.1" />
    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
  </ItemGroup>

  ...

</Project>
```

**Comment by rchande** [comment]
@akshita31 You've been looking at unit testing. Can you take a look?

**Comment by akshita31** [comment]
@firelizzard18 This may be problem with the .debugger folder in your install directory. Could you try deleting the .debugger folder and the install.Lock file in your .vscode/extensions/ms-vscode.csharp-1.13.1 directory and then restarting vscode? (The path to the .vscode directory can be seen in the Omnisharp Log)

**Comment by gregg-miskelly** [comment]
@akshita31 maybe I am missing something, but the crash is in OmniSharp. What do you think might be going wrong with the debugger?

**Comment by gregg-miskelly** [comment]
@firelizzard18 can you debug other things? (ex: create a new console app (`dotnet new console`) and try to debug).

**Comment by firelizzard18** [comment]
@akshita31 I will try to discover what I was working on when I found this error.

@gregg-miskelly I have been able to debug other things in the past. I seem to be getting all kinds of interesting issues when attempting to debug .NET Framework projects. I've reinstalled the extension already, so if the `.debugger` folder was the issue, I can't test that.

**Comment by firelizzard18** [comment]
I've also been switching between Visual Studio and VSCode, as the former occasionally is able to debug my projects.

**Comment by Adam-Pond** [comment]
I found that updating to the 64bit version of VSCode solved the problem. Not sure this was the cause or in doing so, it cleared up some other files or something that was the cause.

**Comment by akshita31** [comment]
@firelizzard18 Can you provide the project where this error occurs, to help debug.

**Comment by firelizzard18** [comment]
@akshita31 The project I was working on is owned by my employer and not open source :(

**Comment by akshita31** [comment]
@firelizzard18 Are you able to perform "Run Test" successfully ? Could you post the Omnisharp log here ?

**Comment by Adam-Pond** [comment]
I can confirm that a colleague who also had this problem solved it by uninstalling 32bit and installing 64bit version of VSCode.

**Comment by akshita31** [comment]
Thanks @Adam-Pond. @firelizzard18 Did you try this solution ?

**Comment by gregg-miskelly** [comment]
No. We always use a 64-bit vsdbg.

**Comment by firelizzard18** [comment]
@akshita31 I'm not sure I can reproduce this issue. My working environment has changed significantly since I created this issue, and OmniSharp-VSCode no longer works at all for me. Once #2171 is merged and #1694 resolved, I'll try again.

**Comment by JoeRobich** [comment]
Closing as a lot has changed with testing and debugging since this issue was opened. Please reopen if this is still a problem and provide updated details.
